### ISSUE TITLE
Image Tapped event not functional

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#3485]
## Current behavior

`Tapped` event on `Image` does not work on Android. Possibly on iOS/macOS as well, WASM works normally.

## Expected behavior

`Tapped` event should fire on interaction.

## How to reproduce it (as minimally and precisely as possible)

Attach handler to `Image.Tapped` event.

```
<Image Source="/Assets/Image.png" Tapped="Image_Tapped" />
```

Code-behind:

```
private void Image_Tapped(object sender, TappedRoutedEventArgs e)
{
    global::System.Diagnostics.Debug.WriteLine("Tapped");
}
```

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
This issue may get fixed by https://github.com/unoplatform/uno/pull/3203#issuecomment-653658244

**Comment by davidjohnoliver** [comment]
> This issue may get fixed by [#3203 (comment)](https://github.com/unoplatform/uno/pull/3203#issuecomment-653658244)

Indeed, made a note to check this

**Comment by davidjohnoliver** [comment]
Update: yes, seems it is indeed fixed by #3203, which makes `Image` a `FrameworkElement`
